### ISSUE TITLE
Allow dependencies to be defined over multiple lines with `\`

### ISSUE THREAD

**Issue by hbjydev** [casey/just#1851]
It'd be really useful to be able to do something like this:

```
deploy-all: \
  (deploy 'us-west-2') \
  (deploy 'us-east-1') \
  (deploy 'eu-west-2')
  echo "Deployed!"

deploy region:
  npm run cdk -- deploy --region='{{region}}'
```

Primarily for 'meta-tasks' where one task might _only_ be a collection of dependencies with arguments. Those end up being pretty long lines currently, so being able to specify multiple lines of dependencies for a task would help the readability of the file.

Thanks for making Just, it's awesome :)

**Comment by laniakea64** [comment]
The exact syntax you're requesting [is already implemented as of `just` 1.15.0](https://just.systems/man/en/chapter_47.html#outside-recipe-bodies).

**Comment by hbjydev** [comment]
Oh! I didn't see that in the docs! Thanks @laniakea64 :)
